hapi-fhir-base/src/main/java/ca/uhn/fhir/util/ObjectUtil.java

```diff
@@ -25,14 +28,12 @@
 
 public class ObjectUtil {
 
+	/**
+	 * Just use Objects.equals() instead;
```

Comment:
```suggestion
	 * @deprecated Just use Objects.equals() instead;
```

I think errorprone will complain if you use the `@Deprecated` annotation without a corresponding `@deprecated` javadoc (and I think that's a good idea anyhow)